test/tests/parallel/set1/spam_virus/clamav.bats

```diff
@@ -26,7 +26,7 @@ function setup_file() {
   wait_for_service "${CONTAINER_NAME}" postfix
   wait_for_smtp_port_in_container "${CONTAINER_NAME}"
 
-  _run_in_container bash -c "nc 0.0.0.0 25 < /tmp/docker-mailserver-test/email-templates/amavis-virus.txt"
+  _run_in_container /bin/bash -c "nc 0.0.0.0 25 < /tmp/docker-mailserver-test/email-templates/amavis-virus.txt"
```

Comment:
Not really, it's just more consistent with the rest of the test cases I looked at.

Reply:
I won't block on this as it's not relevant to the test, although for that same reason since I think that's the only change in that file causing all the diffs, it could be reverted too :man_shrugging: 

After we've converted all the serial tests to leverage the `setup.bash` helper, we can be nitpicky about consistency and test case names I guess :sweat_smile: 

Resolving as non-blocking concern.